recipes/cerbere/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set version = "2.1.0" %}
+
+package: 
+  name: cerbere
+  version: {{ version }}
+
+source:
+  url: https://gitlab.ifremer.fr/cerbere/cerbere/-/archive/V2.1.0/cerbere-{{ version }}.tar.gz
+  sha256: de3aae083cca91381b642a10282c9718849b2a46beda596c67cb8eb8ceb7ebee
+
+build:
+  number: 0
+  skip: true
```

This is skipping all builds. I think this is pure Python and could be noarch?
```suggestion
  noarch: python
```